test/apiTest.js

```diff
@@ -1,58 +1,79 @@
 // next we will write an API test for '/breed routes' GET and GET by id: 
```

Comment:
this is good :) you will probably want to split that up into a separate test file when you do that, but that should be done in a separate PR.